assets/src/components/template-mode-option/index.js

```diff
@@ -136,16 +137,13 @@ export function TemplateModeOption( { children, details, initialOpen, labelExtra
 			selected={ mode === themeSupport }
 		>
 			<div className="template-mode-selection__details">
-				{ details && (
-					<p>
-						<span dangerouslySetInnerHTML={ { __html: details } } />
-						{ ' ' }
-						{ /* @todo Temporary URL. */ }
-						<a href="http://amp-wp.org" target="_blank" rel="noreferrer">
-							{ __( 'Learn more.', 'amp' ) }
-						</a>
-					</p>
-				) }
+				<p>
+					<span dangerouslySetInnerHTML={ { __html: details } } />
+					{ ' ' }
+					<a href={ detailsUrl } target="_blank" rel="noreferrer noopener">
+						{ __( 'Learn more.', 'amp' ) }
+					</a>
```

Comment:
@amedina FYI: The “Learn More” links now go to the respective template mode pages on amp-wp.org rather than all going to the homepage.

![image](https://user-images.githubusercontent.com/134745/109256903-de711d00-77ab-11eb-8cb5-2d9df3092eab.png)

Reply:
Excellent.